datavault/backend.py

```diff
@@ -179,26 +403,25 @@ def _saveData(self, data):
         f.flush()
 
     def addData(self, data):
```

Comment:
The parent addData accepts a single tuple and converts it to a list (l325), this one doesn't - is that intentional?